test/e2e/common/setup-kubestellar.sh

```diff
@@ -109,14 +104,25 @@ esac
 : Install the core-chart
 :
 
-pushd "${SRC_DIR}/../../.."
-make kind-load-image
-make install-local-core-chart \
-  INSTALL_KUBEFLEX=false \
-  ITS_NAME=its1 \
-  DEFAULT_WDS_NAME=wds1 \
-  KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY=$KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY
-popd
+if [ "$use_release" = true ] ; then
+  kubestellar_latest_release=$(curl -sL https://github.com/kubestellar/kubestellar/releases/latest | grep "</h1>" | tail -n 1 | sed -e 's/<[^>]*>//g' | xargs)
```

Comment:
What we want is not the latest release on GitHub but rather the last release up through the commit that the user is using. See https://github.com/kubestellar/kubestellar/blob/5c18a50ccd0bbe7f2124b522a235215b7289bb66/test/e2e/README.md?plain=1#L7 . We do this by putting self-references in files. The list of self-references is tracked in https://docs.kubestellar.io/release-0.24.0/direct/release/#making-a-new-kubestellar-release . We could put a self-reference in this file (e.g., `KS_RELEASE=0.24.0`).

Reply:
While I understand your point, and the complications in digging the version out, I'm wondering if it is in reality moot in a real case scenario.

I was thinking that one would `run_test --released` basically only once after a new release (being alpha, preview, stable, etc) ... at that would be `latest` independently.

Do you envision a situation when one would like to `run_test --released` against v0.24.0 when you release `v0.24.1.alpha1`? In such caases I think we go back to my initial proposal where `--released [<ver>]` could take an optional version.

Finally, I think we should remove `/VERSION` since it is not maintained anymore.